### PR TITLE
ACMS-1989: Update admin theme in Acquia CMS to Gin.

### PR DESCRIPTION
**Pull request by rajeshreeputra**
**Motivation**
Fixes #[ACMS-1989](https://acquia.atlassian.net/browse/ACMS-1989)

**Proposed changes**
This PR is created to check if there any test failure with Gin theme.

**Alternatives considered**
This can be closed without any further action once the issue identification is completed, or it can be used, and further changes can be made here directly.

**Testing steps**
NA

**Merge requirements**
- [ ] _Major change_, _Minor change_, _Bug_, _Enhancement_, and/or _Chore_ label applied
- [ ] Manual testing by a reviewer

### CODE REVIEW

File: modules/acquia_cms_common/acquia_cms_common.install [hunk 1]
```diff
@@ -27,16 +27,16 @@ function acquia_cms_common_install($is_syncing) {
       'view media',
     ]);
 
-    // Enable acquia_claro theme.
+    // Enable gin theme.
     $themesList = \Drupal::service('extension.list.theme')->getList();
     $themesInstall = ["olivero"];
-    if (isset($themesList['acquia_claro'])) {
-      $themesInstall[] = "acquia_claro";
+    if (isset($themesList['gin'])) {
```
Comment (apathak18) [inline modules/acquia_cms_common/acquia_cms_common.install:33]: Can we do something like this?
$themes_install =  $themes_list['gin'] ? ['olivero', 'gin'] : ['olivero'];

File: modules/acquia_cms_common/acquia_cms_common.install [hunk 2]
```diff
@@ -27,16 +27,16 @@ function acquia_cms_common_install($is_syncing) {
...
+      $themesInstall[] = "gin";
     }
     \Drupal::service('theme_installer')->install($themesInstall);
     \Drupal::configFactory()->getEditable('system.theme')->set('default', 'olivero')->save();
-    if (in_array("acquia_claro", $themesInstall)) {
-      \Drupal::configFactory()->getEditable('system.theme')->set('admin', "acquia_claro")->save();
+    if (in_array("gin", $themesInstall)) {
+      \Drupal::configFactory()->getEditable('system.theme')->set('admin', "gin")->save();
```
Comment (apathak18) [inline modules/acquia_cms_common/acquia_cms_common.install:39]: Can we do something like this?
$system_config_factory = \Drupal::configFactory()->getEditable('system.theme');
$system_config_factory->set('default', 'olivero')->save();
if (in_array("gin", $themes_install)) {
$system_config_factory->set('admin', 'gin')->save();
}

**Review by apathak18 — APPROVED** [review]
Added a minor suggestion rest everything looks good

File: modules/acquia_cms_article/tests/src/Functional/BlogTest.php [hunk 3]
```diff
@@ -22,7 +22,7 @@ class BlogTest extends BrowserTestBase {
   /**
    * {@inheritdoc}
    */
-  protected $defaultTheme = 'stark';
+  protected $defaultTheme = 'gin';
```
Comment (chandan-singh7929) [inline modules/acquia_cms_article/tests/src/Functional/BlogTest.php:25]: I don't think we should test this on gin

File: modules/acquia_cms_common/acquia_cms_common.install (hunk 1 above)
Comment (chandan-singh7929) [reply modules/acquia_cms_common/acquia_cms_common.install:33]: @apathak18 - Any reason for this suggestion?

**Review by chandan-singh7929 — CHANGES_REQUESTED** [review]
@rajeshreeputra - I have requested some changes, please take a look.

File: modules/acquia_cms_common/acquia_cms_common.install (hunk 1 above)
Comment (apathak18) [reply modules/acquia_cms_common/acquia_cms_common.install:33]: Doing the same thing with 3-4lines and variable name should be same

Comment (rajeshreeputra) [reply modules/acquia_cms_common/acquia_cms_common.install:36]: @apathak18 requesting re-review.

**Review by apathak18 — APPROVED** [review]

**Review by deepakmishra2 — APPROVED** [review]

**Review by chandan-singh7929 — COMMENTED** [review]
@rajeshreeputra - I can see some changes which are not related to work of this ticket, like refactoring, I would suggest to this in separate ticket and let only put the change which are related to this ticket.

Also don't include patch for the work that is not merged, rather provide steps to pull that in during verification.

**Comment by rajeshreeputra** [comment]
> I can see some changes which are not related to work of this ticket, like refactoring, I would suggest to do this in a separate ticket and lets add only changes which are related to this ticket.
Removed the changes not related to work of this ticket.

>Also don't include patch for the work that is not merged, rather provide steps to pull that in during verification.
The patch has been added to the root composer.json file. In our CI environment, the site is installed using the starterkit, and it's important to note that there have been changes related to the Gin theme, although these changes have not been released yet. This means that our CI process will encounter failures because the starterkit version 1.1.5 now requires Acquia Claro to be installed in order to set up the site and run tests. However, in the pull request, Acquia Claro has been removed, causing the CI failures. Hence it is required to be added in root composer.json file.

**Comment by chandan-singh7929** [comment]
> The patch has been added to the root composer.json file. In our CI environment, the site is installed using the starterkit, and it's important to note that there have been changes related to the Gin theme, although these changes have not been released yet. This means that our CI process will encounter failures because the starterkit version 1.1.5 now requires Acquia Claro to be installed in order to set up the site and run tests. However, in the pull request, Acquia Claro has been removed, causing the CI failures. Hence it is required to be added in root composer.json file.

ok, then lets create a ticket to remove this patch when we do the release for this changes.

**Pull request merged**